.gitlab-ci.yml

```diff
@@ -81,14 +81,12 @@ integrate:
   allow_failure: true
   parallel:
     matrix:
-      - PY_VER: '3.6'
-        REPO_NAME: [data-tools, esbuild, gdcapi, gdcbatch, inspector]
       - PY_VER: '3.7'
-        REPO_NAME: [gdcadmin, sheepdog]
+        REPO_NAME: [gdcadmin, gdcbatch, esbuild, sheepdog]
```

Comment:
I don't think I've seen this before. Is this to trigger downstream builds?

Reply:
yes